src/frontend/apps/desk/src/pages/404.tsx

```diff
@@ -1,25 +1,72 @@
-import { ReactElement } from 'react';
-import { useTranslation } from 'react-i18next';
+import {Button} from '@openfun/cunningham-react';
+import {ReactElement} from 'react';
+import {useTranslation} from 'react-i18next';
 
-import { Text } from '@/components';
-import { NextPageWithLayout } from '@/types/next';
+import {Box, Text, StyledLink} from '@/components';
+import {NextPageWithLayout} from '@/types/next';
 
 import MainLayout from './MainLayout';
 
+import Icon404 from '@/assets/icons/icon-404.svg';
+
+import styled from "styled-components";
+import useCunninghamTheme from "@/cunningham/useCunninghamTheme";
+
+const StyledButton = styled(Button)`
+  width: fit-content;
+  padding-left: 2rem;
+  padding-right: 2rem;
+`;
+
 const Page: NextPageWithLayout = () => {
-  const { t } = useTranslation();
+  const {t} = useTranslation();
+  const {colorsTokens} = useCunninghamTheme()
 
   return (
-    <Text
-      className="mt-xl"
-      as="h1"
-      $justify="center"
+    <Box
       $align="center"
-      $theme="danger"
-      $textAlign="center"
+      className="mr-s ml-s"
+      $css={'margin-top: 115px'}
     >
-      {t('404 - Page not found')}
-    </Text>
+      <div data-testid="icon-404">
+        <Icon404 />
+      </div>
+
+      <Box
+        $align="center"
+        $css={`margin-top: 29px`}
+      >
+        <Text
+          $size="h2"
+          $weight="500"
+          $theme="greyscale"
+          $variation="900"
+        >
+          {t('404 title')}
```

Comment:
The keys are directly in english translated.

```suggestion
          {t('Ouch !')}
```